pkgs/development/python-modules/ring_doorbell/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, pytestCheckHook
+, pytz
+, requests
+, requests-mock
+, requests_oauthlib
+}:
+
+buildPythonPackage rec {
+  pname = "ring_doorbell";
+  version = "0.6.2";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0q35sji2f4lphlr0hdpzazfbi6rrnyadh1k5q9abicr759r3gmgv";
+  };
+
+  propagatedBuildInputs = [
+    pytz
+    requests
+    requests_oauthlib
+  ];
```

Comment:
According to `install_requires` is [`oauthlib`](https://github.com/tchellomello/python-ring-doorbell/blob/master/setup.py#L28) a requirement. `oauthlib` should be mentioned even if `requests_oauthlib` is pulling it in. This avoids issues if `requests_oauthlib` drops `oauthlib`.

Reply:
Fixed, thanks.